### DISCUSSION TITLE
How to check which model is currently in operation?

### DISCUSSION THREAD

**Discussion by riordant** [CopilotC-Nvim/CopilotChat.nvim#398] [Q&A] [answered]
I can use `:CopilotChatModels` to choose a model. I can also set the model in `init.lua`. But is there a way to tell which model is currently in use? 
For example, I could choose any of the available models in the model list, but it may not be activated, depending on my copilot level (free, personal, enterprise). Or, I could enter a model in `init.lua`, but if it's misspelled, I won't get an error on start, so it must fall back to some other model. 
So, a way to see currently active model is needed.

**Comment by gptlang** [comment]
You can't set an invalid model:
```
Failed to get response: {"error":{"message":"The requested model is not supported.","param":"model","code":"model_not_supported","type":"invalid_request_error"}}
```

But indeed, having a way to see which model is in use would be helpful. (todo)

**Comment by gptlang** [comment, marked as answer]
`:CopilotChatModel`
